src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -114,7 +130,8 @@ protected function getDefinedElements()
     {
         return array_merge(parent::getDefinedElements(), [
             'name' => '#sylius-product-name',
-            'attributes' => '#sylius-product-attributes'
+            'attributes' => '#sylius-product-attributes',
+            'out-of-stock-label' => '#sylius-product-out-of-stock',
```

Comment:
All of this elements are labels somehow. `out-of-stock-label` -> `out-of-stock`?